src/main/java/org/example/repository/impl/BookRepositoryImpl.java

```diff
@@ -37,8 +38,12 @@ public List<Book> findAll() {
         try (Session session = factory.openSession()) {
             return session.createQuery("from " + Book.class.getSimpleName(), Book.class)
                     .getResultList();
-        } catch (Exception e) {
-            throw new RuntimeException("Can't get all books from db", e);
```

Comment:
there is should be catch block
Also, in repo layer use DataProcessingException instead of RuntimeException

Reply:
what for catch block?
DataProcessingException - is a custom one? it should be extended from runtime?

Reply:
Correct, if something happened during request to DB it's better to check it and throw an exception